tests/mock_datasource/rubric_level.tfmock.hcl

```diff
@@ -0,0 +1,9 @@
+mock_data "opslevel_rubric_level" {
+  defaults = {
+    alias = "mock-rubric-level-alias"
+    name  = "mock-rubric-level-name"
+    # filter is not set here because its fields are not computed
+    # id intentionally omitted - will be assigned a random string
+    # index intentionally omitted - defaults to number type 0
```

Comment:
It would be better to just set an integer value here rather than `0` so that it is less ambiguous where the value was derived from.